test/controllers/ajax_controller_test.rb

```diff
@@ -568,9 +568,9 @@ def test_name_primer
   def build_name_primer_item(name)
     { id: name.id,
       text_name: name.text_name,
+      author: name.author,
       deprecated: name.deprecated,
-      synonym_id: name.synonym_id,
-      author: name.author }.to_json
+      synonym_id: name.synonym_id }.to_json
```

Comment:
For some reason, Rails 6.1 wants this hash ordered differently now, because the primer is returning results differently.